### PR TITLE
Update OWNERS file

### PR DESCRIPTION
**Pull request by awgreene**
This commit removes a number of engineers from the OWNERS file that no longer work at Red Hat.

Signed-off-by: Alexander Greene <greene.al1991@gmail.com>

### CODE REVIEW

**Comment by awgreene** [comment]
It would be iconic if @exdx, @njhale, or @hasbro17 approved/lgtm'd this PR.

**Comment by hasbro17** [comment]
Since you asked :)
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -4,23 +4,18 @@ approvers:
...
   - anik120
-  - exdx
   - awgreene
-  - hasbro17
   - ankitathomas
   - joelanford
   - timflannagan
   - perdasilva
 reviewers:
-  - njhale
```
Comment (hasbro17) [inline OWNERS:15]: Also line 2

**Comment by timflannagan** [comment]
This is a no-op, so adding the required no-ff labels.

/override ci/prow/e2e-gcp-olm
/label px-approved
/label qe-approved
/label docs-approved

**Comment by timflannagan** [comment]
No need to recycle e2e and waste resources on a no-op.

/override ci/prow/e2e-gcp-console-olm
/override ci/prow/e2e-gcp-ovn
/override ci/prow/e2e-upgrade

**Pull request merged**